DOC, ENH: Add 1.13.0-changelog.rst

charris opened the pull request:
Create a new `doc/changelog/` directory and add the `1.13.0-changelog.rst` there. The idea is to track the changelogs and keep them available, but without attaching them to the release notes.

charris committed:
DOC: Add 1.13.0-changelog.rst

Create a new `doc/changelog/` directory and add the 1.13.0-changelog.rst
there.

rgommers commented:
+1 this belongs in the release notes as sent to mailing lists, but not in the html docs.

charris commented:
@rgommers I'll add changelogs for the 1.12.x releases and prune the 1.12.0 release notes.

charris committed:
MAINT: Add changelogs for 1.12.0 and 1.12.1

Prune the changelog info out of the 1.12.0 1.12.1 release notes and
create corresponding changelog files. This will shorten the generated
documentation.

[ci skip]

rgommers commented:
In it goes, thanks @charris
